config/cluster/host.go

```diff
@@ -240,6 +269,24 @@ func (h *Host) WaitKubeNodeReady(node *Host) error {
 	)
 }
 
+// DrainNode drains the given node
+func (h *Host) DrainNode(node *Host) error {
+	_, err := h.ExecOutput(h.Configurer.KubectlCmdf("drain --grace-period=120 --force --timeout=5m --ignore-daemonsets --delete-local-data %s", node.Metadata.Hostname), exec.HideOutput())
```

Comment:
```suggestion
	err := h.Exec(h.Configurer.KubectlCmdf("drain --grace-period=120 --force --timeout=5m --ignore-daemonsets --delete-local-data %s", node.Metadata.Hostname))
```

* No need to use `ExecOutput` if the output is not used
* `exec.HideOutput()` hides the command output from the debug log, there shouldn't be anything sensitive in drain output.